app/services/workarea/zipco/order.rb

```diff
@@ -94,13 +94,19 @@ def shipping_service
 
         def items
           items_array = order.items.map do |oi|
+           item_image = if Rails.application.config.action_controller.asset_host.present?
```

Comment:
Seems a little weird to only include this is there's a CDN? What if you want to test locally or something...

Reply:
If the asset host is not present then the product_image_url method returns a relative path, which the zip payments endpoint rejects.

Reply:
Hmm ok. Seems better to do a direct check then - if the zip payments endpoint rejects relative URLs, check for that, like `if url.starts_with?('/')`...